src/pages/patientView/structuralVairnt/column/AnnotationColumnFormatter.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+    civicSortValue,
+    DEFAULT_ANNOTATION_DATA,
+    GenericAnnotation,
+    IAnnotation,
+    USE_DEFAULT_PUBLIC_INSTANCE_FOR_ONCOKB,
+    oncoKbAnnotationSortValue,
+    RemoteData,
+} from 'react-mutation-mapper';
+import { CancerStudy, StructuralVariant } from 'cbioportal-ts-api-client';
+import { IAnnotationColumnProps } from 'shared/components/mutationTable/column/AnnotationColumnFormatter';
+import {
+    IOncoKbData,
+    generateQueryStructuralVariantId,
+} from 'cbioportal-frontend-commons';
+import { CancerGene, IndicatorQueryResp } from 'oncokb-ts-api-client';
+
+export default class AnnotationColumnFormatter {
+    public static getData(
+        structuralVariantData: StructuralVariant[] | undefined,
+        oncoKbCancerGenes?: RemoteData<CancerGene[] | Error | undefined>,
+        oncoKbData?: RemoteData<IOncoKbData | Error | undefined>,
+        usingPublicOncoKbInstance?: boolean,
+        uniqueSampleKeyToTumorType?: { [sampleId: string]: string },
+        studyIdToStudy?: { [studyId: string]: CancerStudy }
+    ) {
+        let value: IAnnotation;
+
+        if (structuralVariantData) {
+            let oncoKbIndicator: IndicatorQueryResp | undefined = undefined;
+            let oncoKbStatus: IAnnotation['oncoKbStatus'] = 'complete';
+
+            let oncoKbGeneExist = false;
+            let isOncoKbCancerGene = false;
+            if (oncoKbCancerGenes && !(oncoKbCancerGenes instanceof Error)) {
+                oncoKbGeneExist =
+                    _.find(
```

Comment:
_.some will return boolean and free you from testing against undefined

Reply:
yeah, I tried to copy code from existing annotation file which is used for cna table https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx. I'll update it